packages/core-services/src/events/Events.ts

```diff
@@ -53,6 +53,17 @@ type LoginServiceConfigurationEvent = {
 	  }
 );
 
+// TODO: Decided to keep this here for now, but it should be moved to a more appropriate place in the future
+export interface IMentionCounter {
+	toAll: boolean;
+	toHere: boolean;
+	mentionIds: string[];
+}
+
+export enum EventNames {
+	USER_MENTIONS = 'user.mention',
```

Comment:
Would not it be clearer to have another value for the event, since we are not handling only user mentions through this event, but also `@all` and `@here`?